docs/INSTALL.md

```diff
@@ -607,6 +607,10 @@ Set up your developer account and add certificates:
 - Click "+" and choose "Apple Development".
 - You may also need to register your Mac in your Apple Developer account.
 
+If you want to run Organic Maps on a real device, you need a CarPlay entitlement. You can either request the entitlement for your own
+bundle identifier from Apple (see [here](https://developer.apple.com/documentation/carplay/requesting_carplay_entitlements)) or you join
+the Organic Maps Apple Developer team. Please contact us to request access to our team.
```

Comment:
I remember that it was possible to run OM without CarPlay support by removing entitlement. Did you manage to do that?

Reply:
Ah no I'll try that later and get back to you. Thanks!

Reply:
@biodranik You're right. I remove the entitlement and now I can install Organic Maps on my iPhone if I select a different team. I've updated `INSTALL.md` with the steps to remove the entitlement.